app/services/event_service.rb

```diff
@@ -0,0 +1,11 @@
+class EventService
+  def self.find_by_zip(zipcode)
+    # link to docs http://api.eventful.com/docs/events/search
+    # refactor the faraday.get and params
+    # I think the March piece should really be Next+Month, get it working then try it out
+    response = Faraday.get("http://api.eventful.com/rest/events/search?l=#{zipcode}&t=March+2017&app_key=#{ENV['APP_KEY']}")
```

Comment:
it'd be easier to read and clearer if you set the parameters as variables above and then constructed the url here.

you could also set `Faraday.new` to a variable then add the parameters on another line - this may be the most preferable way (it's easy to read and we could reuse the first piece with different parameters)